src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final int MAX_TASKS = 100;
+
+    public static String getFirstWord(String word){
+        String firstWord = word;
+        if (firstWord.contains(" ")) {
+            firstWord = firstWord.substring(0, firstWord.indexOf(" "));
+        }
+        return firstWord;
+    }
+
+    public static boolean checkOneWord(String word){
+        String [] wordArray = word.trim().split(" ");
+        if(wordArray.length == 1){
+            return true;
+        } else{
+            return false;
+        }
+    }
+    public static void main(String[] args) throws DukeException {
+        Scanner in = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+        Task[] items = new Task[MAX_TASKS];
+        int itemsCount = 0;
+        String input = in.nextLine();
+//        String firstWord = getFirstWord(input);
+        while (true) {
+//            String firstWord = getFirstWord(input);
+//            System.out.println(firstWord);
+            if (input.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            } else if (input.equals("list")){
+                System.out.println("Here are the tasks in your list:");
+                for(int i = 0; i<itemsCount; i++){
+//                    System.out.println(i+1 + ".["+ items[i].getStatusIcon() +"] "+ items[i].description);
+                    System.out.println(String.valueOf(i+1) + ". " + items[i]);
+                }
+                input = in.nextLine();
+            } else if (input.contains("done")){
+                try {
+                    int taskNumber = Integer.parseInt(input.replaceAll("[^0-9]", ""));
+                    items[taskNumber - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println("[" + items[taskNumber - 1].getStatusIcon() + "] " + items[taskNumber - 1].description);
+                    input = in.nextLine();
+                } catch (NumberFormatException e){
+                    System.out.println("Task number cannot be empty");
+                    input = in.nextLine();
+                } catch (NullPointerException e){
+                    System.out.println("Please input a valid number");
+                    input = in.nextLine();
+                }
+            } else {
+//                String [] wordArray = input.trim().split(" ");
+                if (input.contains("todo")){
```

Comment:
Since you refer to these constants multiple times, you may want to use variables to store them.